kolibri_oidc_client_plugin/assets/src/views/OIDCLogin.vue

```diff
@@ -4,10 +4,154 @@
     href="/oidcauthenticate/"
     class="button raised secondary"
   >
-    <file-svg
-      slot="icon"
-      src="./openid.svg"
-    />
```

Comment:
hi just lurking :) @jredrejo hope you're doing well! We still need to have that coffee time.

I was just curious – did the `file-svg` loader get removed? If so, there might be other options available e.g. https://github.com/oliverfindl/vue-svg-inline-loader

Reply:
> hi just lurking :) @jredrejo hope you're doing well! We still need to have that coffee time.
> 

yesss

> I was just curious – did the `file-svg` loader get removed? 

Yes, it _disappeared_ when moving from kolibri-components to KDS

> If so, there might be other options available e.g. https://github.com/oliverfindl/vue-svg-inline-loader

Yep,  I saw it but I thought it was not worth it for just one svg that can be inline. In fact I have checked all the svg in KDS are done that way, including them inline inside a vue component: https://github.com/learningequality/kolibri-design-system/tree/develop/lib/KIcon/precompiled-icons/le so I thought this was both the simplest and most consistent solution.

Reply:
With the 0.16 kolibri build tooling we can also just directly require svg files now as well.

Reply:
Coolio.

Regardless of how it's done, the main thing I remember being frustrated by was that you [can't use CSS](https://css-tricks.com/using-svg/#aa-the-problem-with-both-img-and-background-image) to style the "innards" of SVGs unless its been inlined.